<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>portaldialog</class>
 <widget class="QDialog" name="portaldialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1921</width>
    <height>971</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1921</width>
    <height>1080</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>MV Boli</family>
    <pointsize>22</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Personal Account</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/img/Scre.png</normaloff>:/img/Scre.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#portaldialog{
	background-color: qradialgradient(spread:repeat, cx:0, cy:0.034, radius:0.5, fx:0, fy:0, stop:0.11 rgba(2, 0, 46, 253), stop:1 rgba(226, 170, 255, 255));

}</string>
  </property>
  <widget class="QPushButton" name="minimize">
   <property name="geometry">
    <rect>
     <x>1790</x>
     <y>10</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#minimize{
	background-image: url(:/img/minimize.png);
}</string>
   </property>
   <property name="text">
    <string>=</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exit">
   <property name="geometry">
    <rect>
     <x>1840</x>
     <y>10</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#exit{
	background-color: rgb(0, 85, 0);
	color: rgb(255, 255, 255);
	border-top: 3px solid#4f4f4f; /* Top border */
    border-bottom: 3px solid #4f4f4f; /* Bottom border */
    border-left: 3px solid #4f4f4f; /* Left border */
    border-right: 3px solid #4f4f4f; /* Right border */
}</string>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/img/exit.png</normaloff>:/img/exit.png</iconset>
   </property>
  </widget>
  <widget class="Custom_View" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>128</width>
     <height>128</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>128</width>
     <height>128</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>128</width>
     <height>128</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">
#graphicsView{
	background-color: qradialgradient(spread:repeat, cx:0, cy:0.034, radius:0.5, fx:0, fy:0, stop:0.11 rgba(2, 0, 46, 253), stop:1 rgba(226, 170, 255, 255));
}</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>331</width>
     <height>481</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame{
	background-color: rgb(240, 234, 214);
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>221</width>
      <height>411</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="name_edit"/>
     </item>
     <item>
      <widget class="QLineEdit" name="Birthday_edit"/>
     </item>
     <item>
      <widget class="QLineEdit" name="Email_edit"/>
     </item>
     <item>
      <widget class="QLineEdit" name="Address_edit"/>
     </item>
     <item>
      <widget class="QLineEdit" name="Password_edit"/>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>47</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Birthday</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Email</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Address</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>20</y>
     <width>1331</width>
     <height>281</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Vivaldi</family>
     <pointsize>48</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#label_6{
	
	color: rgb(0, 0, 0);
}</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Добро пожаловать в &lt;/p&gt;&lt;p align=&quot;center&quot;&gt;  библиотеку &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>1341</width>
     <height>271</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Vivaldi</family>
     <pointsize>48</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#label_7{
	
	color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Добро пожаловать в &lt;/p&gt;&lt;p align=&quot;center&quot;&gt;  библиотеку &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>200</y>
     <width>731</width>
     <height>261</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Vivaldi</family>
     <pointsize>48</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#label_8{
color: rgb(0, 0, 0);
}</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;ГУАП&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>200</y>
     <width>731</width>
     <height>261</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Vivaldi</family>
     <pointsize>48</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#label_9{
	color: rgb(255, 255, 255);

}</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;ГУАП&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="search_button">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>400</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#search_button{
	border-top: 3px solid#4f4f4f; /* Top border */
    border-bottom: 3px solid #4f4f4f; /* Bottom border */
    border-left: 3px solid #4f4f4f; /* Left border */
	background-color: rgb(255, 255, 255);
    }

#search_button:hover{
	background-color: rgb(255, 85, 0);
    }</string>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="search_edit">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>400</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>7</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#search_edit{
	border-top: 3px solid#4f4f4f; /* Top border */
    border-bottom: 3px solid #4f4f4f; /* Bottom border */
    border-left: 3px solid #4f4f4f; /* Left border */
    border-right: 3px solid #4f4f4f; /* Right border */
	background-color: rgb(255, 255, 255);
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="show_up_books">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>460</y>
     <width>271</width>
     <height>341</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#show_up_books{
	
	background-color: rgb(240, 234, 214);
	color: rgb(0, 0, 0);
	border-radius:30;
	border-top: 3px solid#4f4f4f; /* Top border */
    border-bottom: 3px solid #4f4f4f; /* Bottom border */
    border-left: 3px solid #4f4f4f; /* Left border */
    border-right: 3px solid #4f4f4f; /* Right border */
}</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>1250</x>
     <y>140</y>
     <width>651</width>
     <height>461</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame{
	border-top: 1px solid#4f4f4f; /* Top border */
    border-bottom: 1px solid #4f4f4f; /* Bottom border */
    border-left: 1px solid #4f4f4f; /* Left border */
    border-right: 1px solid #4f4f4f; /* Right border */
	background-color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QListWidget" name="history_chart">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>321</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="pic_books">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>40</y>
      <width>281</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#label_5{
	color: rgb(0, 0, 0);
}</string>
    </property>
    <property name="text">
     <string>почтовый ящик</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>20</y>
      <width>281</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#label_9{
	color: rgb(0, 0, 0);
}</string>
    </property>
    <property name="text">
     <string>Заимствованные книги</string>
    </property>
   </widget>
   <widget class="QPushButton" name="message_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pie_chart{
	border-top: 3px solid#4f4f4f; /* Top border */
    border-bottom: 3px solid #4f4f4f; /* Bottom border */
    border-left: 3px solid #4f4f4f; /* Left border */
    border-right: 3px solid #4f4f4f; /* Right border */
	border-radius:15px;
	color:black;
	
	background-color: rgb(255, 229, 180);
	
}

#pie_chart:hover{
	color:white;
	background-color: rgb(255, 85, 0);
	
}</string>
    </property>
    <property name="text">
     <string>send</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Message_box_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>400</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_search">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>460</y>
     <width>321</width>
     <height>341</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>240</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Личный кабинет</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>1280</x>
     <y>670</y>
     <width>551</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame_3{
	
	background-color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#label_13{
	
	color: rgb(0, 0, 0);
}</string>
    </property>
    <property name="text">
     <string>Title                      :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#label_14{
	
	color: rgb(0, 0, 0);
}</string>
    </property>
    <property name="text">
     <string>Writer                   :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#label_15{
	
	color: rgb(0, 0, 0);
}</string>
    </property>
    <property name="text">
     <string>Publisher               : </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#label_16{
	
	color: rgb(0, 0, 0);
}</string>
    </property>
    <property name="text">
     <string>Year of Publication :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#label_17{
	
	color: rgb(0, 0, 0);
}</string>
    </property>
    <property name="text">
     <string>Buying a book</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="title_book">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="writer_book">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>90</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="year0fpublication_book">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>170</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="buy_book">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>70</y>
      <width>80</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Buy</string>
    </property>
   </widget>
   <widget class="QLabel" name="display_book">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>150</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#label_18{
	
	color: rgb(0, 0, 0);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="publisher_book">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>130</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Jmax-lim</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ted talks</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Guap</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Te-library</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>UNZA</string>
     </property>
    </item>
   </widget>
  </widget>
  <zorder>minimize</zorder>
  <zorder>exit</zorder>
  <zorder>graphicsView</zorder>
  <zorder>frame</zorder>
  <zorder>label_6</zorder>
  <zorder>label_7</zorder>
  <zorder>label_8</zorder>
  <zorder>label_9</zorder>
  <zorder>search_edit</zorder>
  <zorder>show_up_books</zorder>
  <zorder>search_button</zorder>
  <zorder>frame_2</zorder>
  <zorder>graphicsView_search</zorder>
  <zorder>label_12</zorder>
  <zorder>frame_3</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Custom_View</class>
   <extends>QGraphicsView</extends>
   <header location="global">events.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
